### ISSUE TITLE
Ether not received.

### ISSUE THREAD

**Issue by guzzti** [ethereum/mist#1778]
#### System information
``````
Version: `0.8.9`
OS & Version: windows
Node type: geth 1.5.9
``````

Ok, so I am a huge noob.
I've tried reading a bit around but I haven't managed to grasp my head around this yet.
I transferred money to my wallet on friday, thinking it would arrive as advertized. Transaction can be found here:
https://etherscan.io/address/0xA2C6a12728f69772268BBC8d95fE69d0A0CD187A

However, the Ether doesn't show up in my wallet.
I believe it is not synched, it says on the top of the wallet program: 3,097,950 blocks left, so i think it is downloading something at a dead slow pace.

if I run web3.version i get:
> Object {api: "0.15.3"}

errors arrive at
> web3.admin.nodeInfo.name

resulting in:

> ipc conneciton error
>uncaught error: Unable to connect to socket: timeout at Object.InvalidResponse

Any help will make me happy.

**Comment by darren1975** [comment]
I'm a noob too. I used the "Fund with Bitcoin" function. It showed that it was sent from my Bitcoin wallet and in my Ether wallet it showed awaiting deposit completed and then it started awaiting exchange but I shut down my wallet assuming it would automatically continue the process. When I came back today (Monday), there is nothing in my wallet. Help!

**Comment by lybinhlap** [comment]
I have the same problem. My transaction ID: https://etherscan.io/tx/0x6be0a26c456d07025a6f1ccd354a5fdc281b0a7dc6e6201f98517d6f5e85f703
But now, I have not yet received my Ether on Mist (0.8.9 windows)

**Comment by garretlaxton** [comment]
@Lybinhlap your ether is in your wallet, however your wallet needs to fully sync for the funds to actually show, I would recommend backing up your account for starters. Then let the wallet fully sync which can take up to 1-20 hours, to sync the wallet open the wallet and you will see 'blocks left' below a progression bar and percentage of completeness.

**Comment by garretlaxton** [comment]
Hansi, have you backed up your account yet?

**Comment by hansiiiiiiiiiii** [comment]
yes I did, but what does this help if I haven't received any Ether yet?

**Comment by hansiiiiiiiiiii** [comment]
when I klick refresh Wallet, it shuts down... :-D

**Comment by garretlaxton** [comment]
@hansiiiiiiiiiii You most likely have received, your wallet just hasn't synced to show the funds. Reply with your ETH address & I can confirm if you have received ETH. Once again make sure you have your UTC file saved somewhere safe, you can then try to uninstall an reinstall mist to solve the syncing.

**Comment by hansiiiiiiiiiii** [comment]
i don't want to put my ether adress in Public :-( can I send you a private message?

**Comment by garretlaxton** [comment]
In your browser place your address at the end of this link https://etherscan.io/address/ & you should see your wallets ETH Balance.

**Comment by hansiiiiiiiiiii** [comment]
ok cool, i received 2 Ether. What does "Mined" mean? 

When i buy Bitcoins can I see this Balance  also in this adress?

Thanks a lot!

**Comment by garretlaxton** [comment]
Now to see that in your wallet & to send it, you need to let your wallet fully sync. And this is only an Ethereum wallet, if you 'fund with bitcoin' the BTC you send will be converted into Ethereum. If that's what you where asking.

**Comment by hansiiiiiiiiiii** [comment]
how can I deinstallt the wallet?

**Comment by garretlaxton** [comment]
Make sure your account UTC file is backed up in a safe place & you remember your password else you'll be forever locked out. 
You can then go to `start` type in `run` then in run type `%appdata%` and delete the `Ethereum/Ethereum Wallet` folders, then just install again & import your backed up account.

**Comment by lybinhlap** [comment]
Thank @garretlaxton , I can check and exchange ETH with my wallet by using https://www.myetherwallet.com/.

**Comment by darren1975** [comment]
Cool, I figured mine out too now. I checked the myetherwallet.com site and saw my Ether there. Then I read somewhere that the computer clock has to be correct. I checked mine and it was out 3 minutes so I reset it to the correct time and it started syncing right away. Within 3 hours or so it finished and my ETH showed up. Thanks!

**Comment by APowderBlue** [comment]
@darren1975 Im so glad you posted the time synchronizing thing on the computer because I think we were experiencing the same exact issue. The Wallet wasn't even looking for peers or synchronizing at all.

I went to my computer's time/clock settings > Internet Time tab > Change Settings > selected time.windows.com and hit Update Now button.
I didn't think much of it, but when I went back to the Wallet's window... O_O  There it was, it was connecting with peers and loading blocks.
Currently at only 2% though, but I think I'm heading in the right direction.

Thank you!
